ChangeLog.md

```diff
@@ -1,5 +1,13 @@
 # Revision history for haskell-lsp
 
+## 0.19.0.0 -- 2019-12-14
+
+* Fix vfs line endings (@jneira)
+* Fix typo in .cabal (@turion)
+* Normalize file paths before converting to Uri's (@jneira)
+* Fixes to persistVirtualFile (@mpickering)
+* Fix response type for CodeLensResolve (@SquidDev)
```

Comment:
I think my changes got into the last release, just a little late in the day.

Reply:
Ok, I will check, I was looking at the PR history in github. And I just realised now we did not tag it.